src/smpp_gateway/router.py

```diff
@@ -0,0 +1,46 @@
+from rapidsms.messages.incoming import IncomingMessage
+from rapidsms.messages.outgoing import OutgoingMessage
+from rapidsms.router.blocking import BlockingRouter
+
+from smpp_gateway.models import MTMessage
+
+
+class PriorityIncomingMessage(IncomingMessage):
+    default_priority_flag = MTMessage.PriorityFlag.LEVEL_2
+
+    def respond(self, text, **kwargs):
+        fields = kwargs.get("fields", {})
+        if "priority_flag" not in fields:
+            fields["priority_flag"] = self.default_priority_flag.value
+        kwargs["fields"] = fields
+        return super().respond(text, **kwargs)
+
+
+class PriorityOutgoingMessage(OutgoingMessage):
+    default_priority_flag = MTMessage.PriorityFlag.LEVEL_1
+
+    def extra_backend_context(self):
+        context = super().extra_backend_context()
```

Comment:
Is `self.fields` always populated in this case? Or should we include the same checks from above?